### PR TITLE
Autofac v5 Upgrade

### PR DESCRIPTION
**Pull request by alistairjevans**
Switched to packagereference, changed targeting, updated dependencies.

I double-checked the nuspec constraint on this one!

### CODE REVIEW

File: src/Autofac.Integration.WebApi.Owin/Autofac.Integration.WebApi.Owin.nuspec [hunk 1]
```diff
@@ -14,8 +14,8 @@
     <iconUrl>https://cloud.githubusercontent.com/assets/1156571/13684110/16b8f152-e6bf-11e5-84ae-22c66c6d351a.png</iconUrl>
     <releaseNotes>Release notes are at https://github.com/autofac/Autofac.WebApi.Owin/releases</releaseNotes>
     <dependencies>
-      <dependency id="Autofac" version="[3.5.0,5.0.0)" />
-      <dependency id="Autofac.WebApi2" version="[3.4.0,5.0.0)" />
+      <dependency id="Autofac" version="[5.0.0,6.0.0)" />
+      <dependency id="Autofac.WebApi2" version="[5.0.0,6.0.0)" />
       <dependency id="Autofac.Owin" version="[4.0.0,5.0.0)" />
```
Comment (tillig) [inline src/Autofac.Integration.WebApi.Owin/Autofac.Integration.WebApi.Owin.nuspec]:
```suggestion
       <dependency id="Autofac.Owin" version="[5.0.1,6.0.0)" />
```

**Review by tillig — CHANGES_REQUESTED** [review]
I think there's one line in the .nuspec to update and we're good to go.

File: src/Autofac.Integration.WebApi.Owin/Autofac.Integration.WebApi.Owin.nuspec (hunk 1 above)
Comment (alistairjevans) [reply src/Autofac.Integration.WebApi.Owin/Autofac.Integration.WebApi.Owin.nuspec]: Done; again, thanks for catching it. Should have triple-checked it!

**Comment by alistairjevans** [comment]
I'm not sad that nuspec files aren't really a thing anymore in new projects.

**Review by tillig — APPROVED** [review]
🤖

**Pull request merged**